packages/content/test/attributes/EmailAddress.test.ts

```diff
@@ -0,0 +1,59 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { EMailAddress } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type EMailAddress", function () {
+    test("can create a RepositoryAttribute of Attribute Value Type EMailAddress", function () {
+        const validEMailAdress = EMailAddress.from({
```

Comment:
`validEMailAddress` instead of `validEMailAdress`.

Same in other places below.